web3.md

```diff
@@ -22,6 +22,7 @@ module WEB3
         </blockchain>
         <defaultGasPrice> 20000000000 </defaultGasPrice>
         <defaultGasLimit> 90000       </defaultGasLimit>
+        <lastMineTime>    0           </lastMineTime>
```

Comment:
Should this start at `#time()` instead?

Also, can we call it `lastTime`, instead of `lastMineTime`? It seems that we are using it for other things too, eg. on `#evm_revert` and `#evm_snapshot`, which are not mining.